test/js/node/test/fixtures/rc/duplicate-namespace-option/node.config.json

```
@@ -0,0 +1,6 @@
+{
+    "testRunner": {
+        "test-name-pattern": "first-pattern",
```

Biome (2.1.2)

[error] The key test-name-pattern was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)